packages/fetch/src/types.ts

```diff
@@ -16,182 +16,174 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-export type Method = "GET" | "POST" | "DELETE" | "PUT" | "PATCH";
+export type ResponseType = "basic" | "cors" | "default" | "error" | "opaque" | "opaqueredirect";
+export type RequestCredentials = "include" | "omit" | "same-origin";
+export type RequestMode = "cors" | "navigate" | "no-cors" | "same-origin";
 
-export type Headers = { [name: string]: string };
+export type RequestBody = ReadableStream | string | null; // TODO: Add TypedArray
```

Comment:
What's it take to get a `TypedArray` in here?

Reply:
Defining the `TypedArray` I think. AFAIK it's a web standard.
But .. we're not actively using this from the SDK, so I didn't want to bother.

Reply:
Om wrong, they were introduced in es2017.